Automattic/pocket-casts-android
# Bookmarks - Show bookmark icon on podcast page episode rows

## Description

This displays bookmark icons on podcast page episode rows

## Testing Instructions
- Open a podcast
- Play an episode and add a bookmark
- Return to podcast page 
- ✅ Notice that bookmark icon is shown for that episode
- Download the episode and add to up next 
- ✅ Notice that the bookmark icon is shown at the end of up next and download icons
- Archive the episode
- ✅ Notice that the bookmark icon is shown next to archive icon
- Disable bookmark feature 
- ✅ Notice that bookmark icon is not shown
- Enable bookmark feature 
- Delete the bookmark
- ✅ Notice that bookmark icon is gone


## Screenshots or Screencast 

<img width="300" alt="bookmark" src="https://github.com/Automattic/pocket-casts-android/assets/1405144/18a91cad-ae37-4af7-8f66-6142c68f7301">

<img width="300" alt="bookmark" src="https://github.com/Automattic/pocket-casts-android/assets/1405144/f75067d8-c651-428b-9301-b2beeb72e908"/>


## Checklist
 
#### I have tested any UI changes...
